desc/compute/_basis_vectors.py

```diff
@@ -26,10 +27,13 @@
     transforms={},
     profiles=[],
     coordinates="rtz",
-    data=["B", "|B|"],
+    data=["B", "|B|", "phi"],
+    basis="basis",
 )
 def _b(params, transforms, profiles, data, **kwargs):
     data["b"] = (data["B"].T / data["|B|"]).T
+    if kwargs.get("basis", "rpz").lower() == "xyz":
```

Comment:
rather than having a conditional in every compute function, would it be easier to put it in `desc.compute.compute_utils._compute`?

IE, after computing a qty we check if dim==3 and what the requested basis was?